DemoApp/Sources/Controls/DemoMoreControls/DemoBackgroundEffectSelector.swift

```diff
@@ -0,0 +1,119 @@
+//
+// Copyright © 2024 Stream.io Inc. All rights reserved.
+//
+
+import Foundation
+import StreamVideo
+import StreamVideoSwiftUI
+import SwiftUI
+
+@available(iOS 15.0, *)
+struct DemoBackgroundEffectSelector: View {
+
+    var effects: [BackgroundEffect] = BackgroundEffect.allCases
```

Comment:
Minor: `var` to `let` since we really can't mutate it like this?

(some other uses below as well)